packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -363,13 +365,13 @@ export async function getBuilder(
 
 export function isBundledBuilder(
   parsed: npa.Result,
-  pkg: PackageJson
+  { dependencies = {} }: PackageJson
 ): boolean {
-  if (!parsed.name || !pkg.dependencies) {
+  if (!parsed.name) {
```

Comment:
You could probably remove this `if` statement too 😉

Reply:
No, because `npa('https://example.com')` doesn't contain a `name` property.